common/lib/xmodule/xmodule/modulestore/split_mongo/caching_descriptor_system.py

```diff
@@ -131,6 +131,13 @@ def _load_item(self, usage_key, course_entry_override=None, **kwargs):
 
         class_ = self.load_block_type(block_data.block_type)
         block = self.xblock_from_json(class_, course_key, block_key, block_data, course_entry_override, **kwargs)
+
+        # TODO Once PLAT-1055 is implemented, we can expose the course version
```

Comment:
[PLAT-1055](https://openedx.atlassian.net/browse/PLAT-1055) is now [TNL-5092](https://openedx.atlassian.net/browse/TNL-5092).